static/src/js/components/AccessMethod/AccessMethod.js

```diff
@@ -474,40 +684,43 @@ export class AccessMethod extends Component {
       ? Object.keys(selectedMethod.variables).length > 0
       : false
 
+    const harmonyMethods = accessMethodsByType.Harmony
+
     return (
       <div className="access-method">
         <ProjectPanelSection
-          heading="Select a data access method"
-          intro="The selected access method will determine which customization and output options are available."
+          heading="Choose how you want to download your data"
           step={1}
         >
           <div className="access-method__radio-list">
             {
-              radioList.length === 0
+              radioList.length === 0 && hasHarmony === false
```

Comment:
remove hasHarmony from check

Reply:
Nevermind - `radiolist` only contains the non-Harmony methods now, so we need to check for it being empty as well as the Harmony method type being selected.